### PR TITLE
Stop using usersys registry since it's EOL

### PR DESCRIPTION
**Pull request by pkajaba**

### CODE REVIEW

**Comment by pkajaba** [comment]
Test this

**Comment by pkajaba** [comment]
[test this]

File: Jenkinsfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ node('docker') {
     stage('Build') {
         dockerCleanup()
         docker.build(image.id, '--pull --no-cache .')
-        sh "docker tag ${image.id} docker-registry.usersys.redhat.com/${image.id}"
+        sh "docker tag ${image.id} registry.devshift.net/${image.id}"
```
Comment (msrb) [inline Jenkinsfile]: This won't work: https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.tests#L1

:)

Comment (pkajaba) [inline Jenkinsfile]: Does this apply? 

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.migrations#L1

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/hack/generate-db-migrations.sh#L8

https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/runtest.sh#L15

Comment (msrb) [inline Jenkinsfile]: definitely ;) good catch

**Comment by msrb** [comment]
LGTM, thanks :wink:

**Pull request merged**